packages/components/package.json

```diff
@@ -46,13 +46,14 @@
     "emoji-flags": "^1.2.0",
     "gridicons": "3.1.1",
     "interpolate-components": "1.1.1",
-    "lodash": "^4.17.11",
-    "moment": "^2.22.1",
+    "lodash": "4.17.11",
+    "memoize-one": "5.0.2",
+    "moment": "2.22.1",
     "prop-types": "15.7.2",
-    "qs": "^6.5.2",
-    "react-dates": "^18.0.4",
-    "react-router-dom": "4.3.1",
-    "react-transition-group": "^2.4.0"
+    "qs": "6.7.0",
+    "react-dates": "17.2.0",
```

Comment:
Why the downgrade? Is this to match what Gutenberg is on?

Reply:
Exactly - I don't know if there's still a plan to converge into the Gutenberg datepicker, but keeping on the same version made sense.